apps/examples/lw_aifw_sine_test_app/AICSVDataSource.h

```diff
@@ -19,7 +19,7 @@
 #pragma once
 #include "lw_aifw/lw_aifw_result.h"
 #include "lw_aifw/AIDataSource.h"
-#include "AICSVReader.h"
+#include "lw_aifw/AICSVReader.h"
```

Comment:
why it is moved to app folder?
This is basic class which aifw provide

Reply:
As discussed, I will change